src/components/AboutApp.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+const propTypes = {};
+
+const defaultProps = {};
+
+const AboutApp = () => (
+  <div>
+    {' '}
```

Comment:
To keep your code clean as much as possible, its better to remove unnecessary code.